docs/source/contributing.rst

```diff
@@ -0,0 +1,3 @@
+.. highlight:: sh
```

Comment:
But this would add another level of indirection, making it harder to find the actual information!

Also, it wouldn't work offline (and that's an actual use case for me, as I'm frequently on trains with no network coverage).

Another point why I like to have the actual information in `CONTRIBUTING.rst` is that it will always be the correct version.
If it only contains a link to the "latest" version on RTD, it might be the wrong version.